docker/flags.go

```diff
@@ -37,15 +37,13 @@ func getDaemonConfDir() string {
 }
 
 var (
-	flVersion     = flag.Bool([]string{"v", "-version"}, false, "Print version information and quit")
-	flDaemon      = flag.Bool([]string{"d", "-daemon"}, false, "Enable daemon mode")
-	flDebug       = flag.Bool([]string{"D", "-debug"}, false, "Enable debug mode")
-	flSocketGroup = flag.String([]string{"G", "-group"}, "docker", "Group to assign the unix socket specified by -H when running in daemon mode\nuse '' (the empty string) to disable setting of a group")
-	flLogLevel    = flag.String([]string{"l", "-log-level"}, "info", "Set the logging level (debug, info, warn, error, fatal)")
-	flEnableCors  = flag.Bool([]string{"#api-enable-cors", "-api-enable-cors"}, false, "Enable CORS headers in the remote API")
-	flTls         = flag.Bool([]string{"-tls"}, false, "Use TLS; implied by --tlsverify flag")
-	flHelp        = flag.Bool([]string{"h", "-help"}, false, "Print usage")
-	flTlsVerify   = flag.Bool([]string{"-tlsverify"}, dockerTlsVerify, "Use TLS and verify the remote (daemon: verify client, client: verify daemon)")
+	flVersion   = flag.Bool([]string{"v", "-version"}, false, "Print version information and quit")
+	flDaemon    = flag.Bool([]string{"d", "-daemon"}, false, "Enable daemon mode")
+	flDebug     = flag.Bool([]string{"D", "-debug"}, false, "Enable debug mode")
```

Comment:
Where? The daemon is responsible for creating the socket and serving the API. These arguments don't apply to any other docker subcommands...